pkgs/applications/networking/ssb/patchwork/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, appimage-run, fetchurl, runtimeShell, gsettings-desktop-schemas, gtk3, gobject-introspection, wrapGAppsHook }:
+
+let
+  # latest version that runs without errors
+  # https://github.com/ssbc/patchwork/issues/972
+  version = "3.11.4";
```

Comment:
Have you tried 3.11.5? The issue mentions 3.11.6 as being broken.

Reply:
@asymmetric the issue mentions both 3.11.6 and 3.11.5 as broken https://github.com/ssbc/patchwork/issues/972#issuecomment-476215494

Reply:
Yeah, sorry about that. We'll have a new release out soon, currently just blocked by https://github.com/ssbc/patchwork/issues/978.

Reply:
i think that's a followup

i tried the new release alphas and they didn't even load - we can always tweak this later

Reply:
That's no good, sorry to hear that. If you have time to make an issue, I'd love to help debug and ensure that it works on your system. :+1:

Reply:
sure thing, but 3.11.4 works fine for me, i'd rather treat debugging alphas for a future release as a followup

Reply:
Sorry, I read "broken" and thought it was talking about the security vulnerabilities. There's a security issue in 3.11.4 so I'd recommend upgrading ASAP.